src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class IsoscelesTrapezoid extends Shape {
+
+    private Random rand;
+    private StringBuilder sb = new StringBuilder();
+    private int sideA;
+    private int sideB;
+    private int sideC;
+
+    public IsoscelesTrapezoid(String name) {
+        super(name);
+        rand = new Random();
+        this.sideA = rand.nextInt(10);
```

Comment:
it's not clear why do we use random parameters to init the object. 
In the constructor take ready to use parameters and create a new object based on that. For creating random figures use a separate class FigureProducer

Reply:
done!